UI/src/view/csvReportsWorkList.js

```diff
@@ -388,19 +378,10 @@
     };
 
     var buildJobView = function(job) {
-      var jobView = $('.job-content');
-      jobView.append('' +
-          '<div class="job-content-box">' +
-          '<header id="error-list-header">' + 'CSV-eräajon virhetilanteet: ' + job.fileName +
-          '<a class="header-link" style="cursor: pointer;">Sulje</a>' +
-          '</header>' +
-          '<div class="error-list">' +
-          '</div>'
-      );
-      jobView.find('.header-link').on('click', function(){
-        showImporter();
-      });
-      $('.error-list').html(job.content);
+      return  '<header>' +
+                '<h2> CSV-eräajon virhetilanteet: ' + job.fileName + '</h2>' +
```

Comment:
In requirement is Finnish marked as an only required language? There is no need for multilanguage support?

Reply:
@sasuolanderSito 
No, there is no multiple languages support.
It was never a requirement. And it is only used by finnish administrative authority (or something similar).